tests/test_onnx/test_head.py

```diff
@@ -0,0 +1,319 @@
+import mmcv
+import numpy as np
+import torch
+import torch.nn as nn
+from utils import verify_model
+
+from mmdet.models.dense_heads import RetinaHead, YOLOV3Head
+
+# Control the forward of the Test class
+test_step_names = {
+    'forward_single': 0,
+    'forward': 1,
+    '_get_bboxes_single': 2,
+    'get_bboxes': 3
+}
+
+data_path = 'tests/test_onnx/data/'
+
+
+class AnchorHeadTest(nn.Module):
```

Comment:
Please update to a series of simple unit tests of functions and methods of classes.